src/libYARP_manager/src/primresource.cpp

```diff
@@ -166,6 +166,7 @@ Processor::Processor(void) : GenericResource("Processor")
     cpuload.loadAverage1 = (double)0.0;
     cpuload.loadAverage5 = (double)0.0;
     cpuload.loadAverage15 = (double)0.0;
+    cpuload.loadAverageInstant = (double)0.0;
```

Comment:
Sorry, I never noticed this before... These `(double)` casts do not make sense... 0.0 is a double, see http://www.cplusplus.com/doc/tutorial/constants/
@Nicogene I think these should be removed after merging this patch, what do you think?

Reply:
Good point @drdanz , I agree that we should remove this explicit cast